types/parsecurrency/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for parsecurrency 0.2
+// Project: https://github.com/mktj/parsecurrency#readme
+// Definitions by: Cristian Greco <https://github.com/cristianrgreco>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface ParsedCurrency {
+    raw: string;
+    value: number;
+    integer: string;
+    decimals: string;
+    currency: string;
+    symbol: string;
+    decimalSeparator: string;
+    groupSeparator: string;
+}
+
+export default function parsecurrency(currency: string): ParsedCurrency;
```

Comment:
It's not `export default`: https://github.com/mktj/parsecurrency/blob/master/index.js#L5

Reply:
@plantain-00 It's the commonjs equivalent of a default export

Reply:
They are not equivalent: https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default